src/util/mod.rs

```diff
@@ -369,6 +369,23 @@ impl<L, R> Either<L, R> {
     }
 }
 
+/// `build_info` returns a tuple of Strings that contains build utc time and commit hash.
+fn build_info() -> (String, String) {
+    let raw = include_str!(concat!(env!("OUT_DIR"), "/build-info.txt"));
```

Comment:
Yes, we can, but not with build.rs, pass date and hash as environment variables, then call `env!()`.